docs/auth/getting-started.md

```diff
@@ -30,7 +30,7 @@ export class AppComponent {
   constructor(public afAuth: AngularFireAuth) {
   }
   login() {
-    this.afAuth.auth.signInWithPopup(new firebase.auth.GoogleAuthProvider());
+    this.afAuth.auth.signInWithPopup(new firebase.default.auth.GoogleAuthProvider());
```

Comment:
It seems the original code is correct already.

https://firebase.google.com/docs/reference/js/firebase.auth.GoogleAuthProvider

Reply:
With the latest version of the typings and the sdk, it says it cannot find auth. If I use `default` as shown, it not only compiles, but also runs fine.

Reply:
Hmmmm, strange.